server/liveblog/embed/templates/embed.html

```diff
@@ -1,64 +1,33 @@
 <!doctype html>
 <html ng-app="liveblog-embed">
     <head>
-        <title>Embed</title>
+        <title>{{ blog.title }}</title>
```

Comment:
It's useful for developers when you open the page in the browser, and for SEO.

You mean something like that http://npr.github.io/responsiveiframe/ ?